frontend/src/scenes/data-management/events/EventDefinitionsTable.tsx

```diff
@@ -41,6 +41,18 @@ const eventTypeOptions: LemonSelectOptions<CombinedEventType> = [
         icon: <UnverifiedEvent />,
         'data-attr': 'event-type-option-event',
     },
+    {
```

Comment:
If its not too much work, it'd be nice to use a divider or indent these sub types since these extra entries could be confusing. Otherwise +1 to hiding this in the UX for this PR and adding in a smaller future one